api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -106,7 +109,11 @@ public void checkFreeTierBillingUsage() {
     final Set<DbUser> newlyExpiredUsers =
         Sets.difference(currentExpiredUsers, previouslyExpiredUsers);
     for (final DbUser user : newlyExpiredUsers) {
-      notificationService.alertUserFreeTierExpiration(user);
+      try {
+        mailService.alertUserFreeTierExpiration(user);
+      } catch (final MessagingException e) {
+        logger.log(Level.WARNING, e.getMessage());
```

Comment:
Sending email alerts is best-effort by design: no retries.

I could report this in a louder or better way if you prefer.